packages/core-data/src/entity-provider.js

```diff
@@ -70,10 +70,10 @@ export default function EntityProvider( { kind, type, id, children } ) {
  * provided entity of the specified type.
  *
  * @param {string} kind The entity kind.
- * @param {string} type The entity type.
+ * @param {string} name The entity name.
  */
-export function useEntityId( kind, type ) {
-	return useContext( getEntity( kind, type ).context );
+export function useEntityProviderId( kind, name ) {
```

Comment:
@adamziel It looks like this rename was a breaking change. There's an example of some breakage in https://github.com/WordPress/gutenberg/pull/39676.

Reply:
Good catch. Yup I can confirm this broke the Navigation block so it will have impact further down the chain.

@adamziel is AFK so I will look into restoring the original and adding a deprecation notice to point to `useEntityProviderId`

Reply:
Follow up here https://github.com/WordPress/gutenberg/pull/39681

Reply:
Also an alternative approach here https://github.com/WordPress/gutenberg/pull/39683

Reply:
Great catch @talldan, I must have missed the fact this was a public API. Thanks for the prompt reaction and a fix!